charts/stable/uptime-kuma/values.yaml

```diff
@@ -20,6 +20,17 @@ service:
         port: 3001
         targetPort: 3001
 
+  # Metrics Services
+  metrics:
+    enabled: true
+    type: ClusterIP
+    ports:
+      metrics:
+        enabled: true
+        protocol: HTTP
+        port: 3002
+        targetPort: 3002
```

Comment:
There are some annotations and also a metrics section that need to be added both in values.yaml and questions.yaml
(Check either block or authentik for examples)

Reply:
I thought this was the only bit missing as metrics was added to Kuma previously but never worked. I'll double check if I missed anything though.

Reply:
Oh, my bad, didn't know there was the other parts there.

Reply:
It might need the annotations on the common.yaml tho

Reply:
Added them now, I think...